custom/opm/health_status.py

```diff
@@ -187,7 +188,10 @@ class AWCHealthStatus(object):
     # our hearts' content.  This would allow us to override definitions of
     # indicators based on their meanings in THIS report.
     def __init__(self, cases):
-        self.cases = cases
+        # Some of the cases are second or third children of the same mother
+        # include that distinction here
+        self.all_cases = cases
```

Comment:
Yeah, exactly.  These aren't actually cases, they're pre-filtered OPMCaseRows, which sort of act as a proxy for cases (with most of the indicators already pre-calculated).